docs/application/vscode-ext/index.md

```diff
@@ -38,17 +38,48 @@ Visual Studio Code Extension for Tizen enables you to develop Tizen .NET and Tiz
 
 Tizen Emulator for VS Code Extension for Tizen has the same requirements as the emulator in Tizen Studio. To check the detailed hardware and software requirements for Tizen Emulator, see [Emulator Requirements](../tizen-studio/setup/prerequisites.md#emulator).
 
+### Option 1
+
 - Intel&reg; Hardware Acceleration Execution Manager (Intel&reg; HAXM) speeds up the Tizen emulation on Intel-VT-enabled systems. For more information, see [Hardware Accelerated Execution Manager](../tizen-studio/setup/hardware-accelerated-execution-manager.md).
 
+> [!NOTE]
+> This option will not applicable and work on AMD processors.
+
 - Make sure **Hyper-V** is disabled (in Windows 10 or higher):
   1. Input **Control Panel** on the **Search** box in Windows 10.
 
   2. Click **Control Panel > Programs and Features > Turn Windows features on or off**.
 
   3. Disable **Hyper-V** and click **OK**.
 
-     ![Disable Hyper-V](media/cs_prerequisite-disable-hiperv.png)
+     Additional note: make sure **Virtual Machine Platform** should be disabled as well to use HAXM.
+
+     ![Disable Hyper-V](media/cs_prerequisite-disable-hyperv.png)
 
   4. Reboot the computer.
 
+### Option 2
+
+- Microsoft's Hyper-V and the Windows Hypervisor Platform (WHPX). Hyper-V is a virtualization feature of Windows that makes it possible to run virtualized computer systems on a physical host computer.
+
+- Make sure **Hyper-V** is enabled (PowerShell in Windows 10 or higher):
+  1. Check the configuration
+
+     > Get-WindowsOptionalFeature -FeatureName Microsoft-Hyper-V-All -Online
+
+     FeatureName      : Microsoft-Hyper-V-All
+     DisplayName      : Hyper-V
+     Description      : Provides services and management tools for creating and running virtual machines and their
+                        resources.
+                        RestartRequired  : Possible
+                        State            : Disabled
+                        CustomProperties :
```

Comment:
This would look better on the site.
```suggestion
    > FeatureName      : Microsoft-Hyper-V-All
    > DisplayName      : Hyper-V
    > Description      : Provides services and management tools for creating and running virtual machines and their resources.
     >                  RestartRequired    : Possible
     >                  State                     : Disabled
     >                  CustomProperties :
```